doc/src/manual/getting-started.md

```diff
@@ -25,8 +25,8 @@ julia> ans
 3
 ```
 
-To exit the interactive session, type `^D` -- the control key together with the `d` key or type
-`quit()`. When run in interactive mode, `julia` displays a banner and prompts the user for input.
+To exit the interactive session, type `CTRL-D` (press the Control/`^` key together with the `d` key), or type
```

Comment:
I appreciate the more explicit phrasing, but I wonder what benefit mentioning the `^` key brings here. I thought the [caret notation](https://en.wikipedia.org/wiki/Caret_notation) was purely symbolic; are there input systems with an actual "^" key used for producing control characters?

FWIW, I actually just tried literally typing `^D` (`^`, `Space`, `Shift`+`D`), `^d` (`^`, `Space`, `D`), and combinations I expected to produce `̂D` (`^`, `Shift`+`D`) and `̂d` (`^`, `D`) in the julia prompt (v0.6.0 if it matters), as well as (`^`+`Shift`+`D`), and none of this worked. For context, I'm using a Portuguese keyboard layout which has a `̂~` key that produces combined characters like "ã" and "â" (using shift), unless a space is typed after the caret key, in which case I get "~a" and "^a". Perhaps @stevengj can provide commentary here regarding whether my input methods may have interfered.

Reply:
I think it's always 'ctrl' (or 'control') on Mac keyboards, but in menus and documentation people nearly always use ⌃ (`U+2303`) or something similar, but that could be mistaken for an up arrow. You can get fonts to show keys, but I wouldn't know whether that's a good idea to rely on it. Images would be a bit more explicit: not ![e2a7f0ab87a1ac02820623e18720a689](https://user-images.githubusercontent.com/52289/37342965-1477213e-26be-11e8-9d39-651ee1e1691c.png) but ![66514edcffc7bbe0570847902c3f5c4b](https://user-images.githubusercontent.com/52289/37342967-149116c0-26be-11e8-9e6b-f212367ac76b.png) but keyboards vary a lot...

Reply:
Just to make sure we're on the same page: are you saying that (to your knowledge) there are indeed physical "^" keys in keyboards --perhaps older ones--, or that there are not, but people tend to represent the control character in documentation as if such keys existed?

Reply:
1: [Yes](http://static1.uk.businessinsider.com/image/5a4d08bdec1ade56f430e91a/people-are-selling-apples-space-gray-imac-pro-accessories-for-over-1000-on-ebay.jpg)

2: they do.

Reply:
Ah, got it. Thanks for clarifying.